vkcs/dc/resource_conntrack_helper.go

```diff
@@ -0,0 +1,310 @@
+package dc
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/clients"
+	conntrackhelpers "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/dc/v2/conntrack_helpers"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/networking"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/util"
+)
+
+// Ensure the implementation satisfies the desired interfaces.
+var _ resource.Resource = &ConntrackHelperResource{}
+
+func NewConntrackHelperResource() resource.Resource {
+	return &ConntrackHelperResource{}
+}
+
+type ConntrackHelperResource struct {
+	config clients.Config
+}
+
+func (r *ConntrackHelperResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = "vkcs_dc_conntrack_helper"
+}
+
+type ConntrackHelperResourceModel struct {
+	ID          types.String `tfsdk:"id"`
+	DCRouterID  types.String `tfsdk:"dc_router_id"`
+	Protocol    types.String `tfsdk:"protocol"`
+	Port        types.Int64  `tfsdk:"port"`
+	Helper      types.String `tfsdk:"helper"`
+	Name        types.String `tfsdk:"name"`
+	Description types.String `tfsdk:"description"`
+	CreatedAt   types.String `tfsdk:"created_at"`
+	UpdatedAt   types.String `tfsdk:"updated_at"`
+	Region      types.String `tfsdk:"region"`
+}
+
+func (r *ConntrackHelperResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:    true,
+				Description: "ID of the resource",
+			},
+
+			"dc_router_id": schema.StringAttribute{
+				Required:    true,
+				Description: "Direct Connect Router ID. Changing this creates a new resource",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+
+			"helper": schema.StringAttribute{
+				Required:    true,
+				Description: "Helper type. Must be one of : \"ftp\".",
```

Comment:
should not we add defaults for this and below one? need to ask dc owners

Reply:
After discussion with dc owners, decided not to add defaults